data/actionsmap/yunohost.yml

```diff
@@ -535,6 +535,31 @@ app:
                     full: --file
                     help: Folder or tarball for upgrade
 
+        ### app_change_url()
+        change_url:
```

Comment:
Hm, dunno, I guess if it's not too much work then why not. But I'm more a core dev than and app packager so I don't care as much about the script name xD